test/unit/jsTest.js

```diff
@@ -0,0 +1,234 @@
+/* eslint-env mocha */
+
+const assert = require('assert')
+const fs = require('fs')
+const fse = require('fs-extra')
+const path = require('path')
+const cleanupTestApp = require('../util/cleanupTestApp')
+const generateTestApp = require('../util/generateTestApp')
+const klawSync = require('klaw-sync')
+const fork = require('child_process').fork
+
+describe('JavaScript Section Test', function () {
+  const appDir = path.join(__dirname, '../app/jsTest')
+  const test1 = 'var a = 7'
+  const test2 = 'var b = "turkey"'
+  const test3 = 'var c = true'
+  let staticDirname = 'js'
+  let pathsOfStaticJS = [
+    path.join(appDir, 'statics', 'js', 'a.js'),
+    path.join(appDir, 'statics', 'js', 'b.js'),
+    path.join(appDir, 'statics', 'js', 'c.js')
+  ]
+  let pathsOfCompiledJS = [
+    path.join(appDir, 'statics', '.build', 'js', 'a.js'),
+    path.join(appDir, 'statics', '.build', 'js', 'b.js'),
+    path.join(appDir, 'statics', '.build', 'js', 'c.js')
+  ]
+  let staticJSFiles = [
+    test1,
+    test2,
+    test3
+  ]
+
+  // function to compile the static folder and files
+  let generateStaticFolder = () => {
+  // make sure the static js directory is there
+    fse.ensureDirSync(path.join(appDir, 'statics', `${staticDirname}`))
+  // write up the differnt files
+    for (let x = 0; x < pathsOfStaticJS.length; x++) {
+      fs.writeFileSync(pathsOfStaticJS[x], staticJSFiles[x])
+    }
+  }
+
+  afterEach(function (done) {
+    cleanupTestApp(appDir, (err) => {
+      if (err) {
+        throw err
+      } else {
+        done()
+      }
+    })
+  })
+
+  it('should compile all static js files using roosevelt-uglify', function (done) {
+    generateStaticFolder()
+    // create and init the app
+    generateTestApp({
+      appDir: appDir,
+      generateFolderStructure: true,
+      suppressLogs: {
+        httpLogs: true,
+        rooseveltLogs: true,
+        rooseveltWarnings: true
+      },
+      js: {
+        compiler: {
+          nodeModule: 'roosevelt-uglify',
+          showWarnings: false,
+          params: {}
+        },
+        whitelist: ['a.js', 'b.js', 'c.js']
```

Comment:
There's no need to specify a whitelist outside of the whitelist test. By default all files in `js` are compiled. The whitelist is used when you want to only compile specific files.